aiplatform/api/AIPlatform.Samples/PredictTextSentimentSample.cs

```diff
@@ -0,0 +1,115 @@
+        // This client only needs to be created once,
+        // and can be reused for multiple requests.
+        var client = new PredictionServiceClientBuilder
+        {
+            Endpoint = $"{locationId}-aiplatform.googleapis.com"
+        }.Build();
+
+        // Configure the parent resource.
+        var endpoint = EndpointName.FromProjectLocationPublisherModel(projectId, locationId, publisher, model);
+
+        // Initialize request argument(s).
+        var content = @"I had to compare two versions of Hamlet for my Shakespeare
+class and unfortunately I picked this version. Everything from the acting
+(the actors deliver most of their lines directly to the camera) to the camera
+shots (all medium or close up shots...no scenery shots and very little back
+ground in the shots) were absolutely terrible. I watched this over my spring
+break and it is very safe to say that I feel that I was gypped out of 114
+minutes of my vacation. Not recommended by any stretch of the imagination.
+Classify the sentiment of the message: negative
+
+Something surprised me about this movie - it was actually original. It was
+not the same old recycled crap that comes out of Hollywood every month. I saw
+this movie on video because I did not even know about it before I saw it at my
+local video store. If you see this movie available - rent it - you will not
+regret it.
+Classify the sentiment of the message: positive
+
+My family has watched Arthur Bach stumble and stammer since the movie first
+came out. We have most lines memorized. I watched it two weeks ago and still
+get tickled at the simple humor and view-at-life that Dudley Moore portrays.
+Liza Minelli did a wonderful job as the side kick - though I'm not her
+biggest fan. This movie makes me just enjoy watching movies. My favorite scene
+is when Arthur is visiting his fiancée's house. His conversation with the
+butler and Susan's father is side-spitting. The line from the butler,
+""Would you care to wait in the Library"" followed by Arthur's reply,
+""Yes I would, the bathroom is out of the question"" is my NEWMAIL
+notification on my computer.
+Classify the sentiment of the message: positive
+
+This Charles outing is decent but this is a pretty low-key performance. Marlon
+Brando stands out. There's a subplot with Mira Sorvino and Donald Sutherland
+that forgets to develop and it hurts the film a little. I'm still trying to
+figure out why Charlie want to change his name.
+Classify the sentiment of the message: negative
+
+Tweet: The Pixel 7 Pro, is too big to fit in my jeans pocket, so I bought new
+jeans.
+Classify the sentiment of the message: ";
+
+        var instances = new List<wkt::Value>
+        {
+            wkt::Value.ForStruct(new()
+            {
+                Fields =
+                {
+                    ["content"] = wkt::Value.ForString(content),
+                }
+            })
+        };
+
+        var parameters = wkt::Value.ForStruct(new()
```

Comment:
Can we add a link or comment that what parameter means what ?

Reply:
I can link to docs but do we usually link to docs from samples?

Reply:
These samples will be embedded in the docs with explanations, so I'm less inclined to add more explanation or links to docs.

Reply:
If these are well explained in the docs where the sample is shown, then that's fine, we don't need extra explanations.

@anuragsrivstv we have data to show that users hardly come to Github straight to look at samples. They usually discover the samples that are included in docs, and then maybe navigate to Github to see the full code, etc.